### PR TITLE
Added section for Wii U guide to switch between Nintendo Network and Pretendo Network profiles

### PR DESCRIPTION
**Pull request by ExperiencersInternational**
I haven't messed with SaveMii yet, so I'm not providing information for that yet. Sorry!

This pull request adds an additional section to the guide for Wii U consoles that explains how to switch between connecting to the Nintendo Network and Pretendo Network.

Probably needs some work doing though.

Solves #228

### CODE REVIEW

**Comment by gitlimes** [comment]
Hi @ExperiencersInternational, thank you for taking time to contribute to Pretendo Network!

I have taken the liberty to rewrite some sentences in your contributions, and I also removed some of the plugins from the Aroma plugin menu, to make it easier to parse visually.

Please let me know what you think about my changes!

**Review by gitlimes — APPROVED** [review]

**Comment by gitlimes** [comment]
@jonbarrow, also let me know what you think

File: docs/en_US/install/wiiu.md [hunk 1]
```diff
@@ -90,3 +92,26 @@ Right before registering a PNID, please make sure to save your password on some
...
+
+Occasionally, you may want to switch between your Pretendo Network and your Nintendo Network profiles on your Wii U; fortunately, it is super simple to switch between the two.
+
+While connected to Pretendo, in the HOME Menu, switch to your Nintendo Network profile. If prompted for a password, do not provide it (if you don't have auto sign-in enabled, you may not have the option to not provide it). You are now inside your Nintendo Network profile.
+
+On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:
+
+<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
+
+Your menu might display additional plugins that aren't shown in this image; however, our focus is specifically on the 'Inkay' plugin, which patches requests intended for Nintendo Network and reroutes them to Pretendo Network instead. Once you've found the entry for Inkay, press `A`.
```
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:106]: I'm gonna use this paragraph as an example but this comment goes for basically the whole page

While I do enjoy having a fun/"personal" vibe with our brand generally, when it comes to our official documentation I do prefer to keep it as short, concise, and professional as possible. That will likely make the documentation come off as "corpo", but there are tons of people who won't understand the instructions unless it's handed to them in the simplest way possible

Doing things this way avoids sections that get too wordy with no real benefit to the user, and gets directly to the point of the section

There's no hard set of rules or guidelines for this though. I kinda just judge if based off vibes. Like for this section I would suggest something like

```suggestion
You may have additional plugins in your menu. Navigate to the `Inkay` plugin and press `A` to enter it's settings.
```

This removes a good portion of fluff from this section, mostly related to the fact that it's already clear what Inkay does based on information earlier in the page. That doesn't need to be restated

**Review by jonbarrow — CHANGES_REQUESTED** [review]
Generally pretty good. Just some suggestions to remove some fluff

File: docs/en_US/install/wiiu.md (hunk 1 above)
Comment (gitlimes) [reply docs/en_US/install/wiiu.md:106]: Ok that's also on me. whoops

File: docs/en_US/install/wiiu.md [hunk 2]
```diff
@@ -90,3 +92,26 @@ Right before registering a PNID, please make sure to save your password on some
...
 
 Finally, set up your PNID, email and password. Once all of this is done, you will be able to use Pretendo servers.
+
+# Switching between Pretendo Network and Nintendo Network
+
+Occasionally, you may want to switch between your Pretendo Network and your Nintendo Network profiles on your Wii U; fortunately, it is super simple to switch between the two.
+
+While connected to Pretendo, in the HOME Menu, switch to your Nintendo Network profile. If prompted for a password, do not provide it (if you don't have auto sign-in enabled, you may not have the option to not provide it). You are now inside your Nintendo Network profile.
+
+On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:
```
[the remark this answers is not in the document]
Comment (gitlimes) [reply docs/en_US/install/wiiu.md:102]: Yeah but you're going to have to switch to your NNID to be able to connect to Nintendo Network

File: docs/en_US/install/wiiu.md (hunk 1 above)
Comment (jonbarrow) [reply docs/en_US/install/wiiu.md:106]: It's not just your changes I actually looked at this PR when it was made and wanted to say the same thing then, I just didn't actually make a review at the time

File: docs/en_US/install/wiiu.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonbarrow) [reply docs/en_US/install/wiiu.md:102]: The reboot that happens after disabling the patches does an actual console soft reboot, you get sent back to the profile selection screen. So selecting a profile does not do anything, as you will leave the profile the moment you enable/disable the patches

[the remark this answers is not in the document]
Comment (ExperiencersInternational) [reply docs/en_US/install/wiiu.md:102]: I don't remember being taken back to that screen personally but I have my Wii U set to my NNID as default

[the remark this answers is not in the document]
Comment (ExperiencersInternational) [reply docs/en_US/install/wiiu.md:102]: I don't really find it's a big issue anyways having that in but if you think that should be changed lmk

File: docs/en_US/install/wiiu.md [hunk 3]
```diff
@@ -90,3 +92,26 @@ Right before registering a PNID, please make sure to save your password on some
...
+You should now be greeted with a menu displaying two buttons. Click the one that says 'Patching'.
+
+To use Nintendo's servers, set 'Connect to the Pretendo Network' to **false**. It should look something like this:
+
+<img src="/assets/images/docs/install/wiiu/pnid/inkay-patching.jpg" width="100%">
+
+Once you've done that, press the `B` button twice, then press the `HOME` button. Your console will reboot with Pretendo Network patches disabled. You can check this by verifying whether a message saying 'Using Nintendo Network' is displayed on boot.
+<img src="/assets/images/docs/install/wiiu/pnid/nintendo-connection.jpg" width="100%">
+
+After that, you should be able to access Nintendo Network. To go back to Pretendo, you can follow these steps again, this time signing into your Pretendo profile and ensuring that 'Connect to the Pretendo Network' is set to **true**.
```
Comment (gitlimes) [inline docs/en_US/install/wiiu.md:117]: A de-fluffed version could be this

```suggestion
# Switching between Pretendo Network and Nintendo Network

To switch between your Pretendo Network and Nintendo Network profiles on your Wii U:

Press `L + Down + SELECT` on the Wii U GamePad to open the Aroma plugin menu.

<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">

Locate and select 'Inkay', then choose 'Patching'. Now, set 'Connect to the Pretendo Network' to **false**.

<img src="/assets/images/docs/install/wiiu/pnid/inkay-patching.jpg" width="100%">

Press `B` twice, followed by the `HOME` button. The console will reboot with Pretendo Network patches disabled, as confirmed by the 'Using Nintendo Network' message.

Now, switch to your desired Network ID via the Account selection screen.

To return to Pretendo Network, repeat the process ensuring 'Connect to the Pretendo Network' is set to **true**.
```

Comment (gitlimes) [reply docs/en_US/install/wiiu.md:117]: @jonbarrow

Comment (jonbarrow) [reply docs/en_US/install/wiiu.md:117]: lgtm

**Comment by gitlimes** [comment]
Do you mind if I commit that lol

**Comment by gitlimes** [comment]
I'm gonna merge this #230 instead, if it's alright with everyone

**Comment by ExperiencersInternational** [comment]
Yeah that's fine lol, at a quick glance there's no real difference

**Comment by gitlimes** [comment]
Closed in favor of #230.

@ExperiencersInternational, thank you again for your contribution!
